frontend/integration-tests/views/overview.view.ts

```diff
@@ -8,8 +8,8 @@ export const detailsSidebarOverviewTab = detailsSidebar.element(by.buttonText('O
 export const detailsSidebarResourcesTab = detailsSidebar.element(by.buttonText('Resources'));
 export const detailsSidebarTitle = $('.resource-overview__heading .co-m-pane__name');
 
-export const itemsAreVisible = () => {
-  return browser.wait(until.presenceOf($(projectOverviewItemSelector)));
+export const itemsAreVisible = async() => {
```

Comment:
These are equivalent, no? Both functions return a promise...

Reply:
Yes, you're right, thank you. I've been looking at these tests too long trying to figure out this flake... :(